template.yaml

```diff
@@ -1,16 +1,42 @@
 AWSTemplateFormatVersion: '2010-09-09'
 Transform: AWS::Serverless-2016-10-31
 Description: >
-  sam-app
-  
-  Sample SAM Template for sam-app
+  CollAction backend
 
 # More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
 Globals:
   Function:
-    Timeout: 5
+    Timeout: 10
+    Environment:
+      Variables:
+        LOCAL_RESOURCES_ONLY: !Ref LocalResourcesOnly
+
+Parameters:
+  Stage:
+    Type: String
+    Default: dev
+    AllowedValues:
+      - dev
+      - prod
+  LocalResourcesOnly:
+    Type: String
+    Default: 'false'
+    AllowedValues:
+      - 'true'
+      - 'false'
 
 Resources:
+
+  HttpApi:
+    Type: AWS::Serverless::HttpApi
+    Properties:
+      StageName: !Ref Stage
+      CorsConfiguration:
+        AllowMethods:
+          - GET
+        AllowOrigins:
+          - http://localhost:8080
+
   HelloWorldFunction:
     Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
     Properties:
```

Comment:
On windows I have to remove the property `Architectures`, otherwise it get this error:
```
InvalidSamDocumentException(
samcli.commands.validate.lib.exceptions.InvalidSamDocumentException: [InvalidResourceException('HelloWorldFunction', 'property Architectures not defined for resource of type AWS::Serverless::Function')] ('HelloWorldFunction', 'property Architectures not defined for resource of type AWS::Serverless::Function')
```
Is this required on MacOS? If so, we can merge for now and I'll find a solution for it later.